Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/DriverPool.hs

```diff
@@ -635,7 +635,8 @@ filterOutGoHomeDriversAccordingToHomeLocation randomDriverPool CalculateGoHomeDr
           keepHiddenForSeconds = Seconds 0,
           goHomeReqId = Just ghrId,
           isForwardRequest = False,
-          previousDropGeoHash = Nothing
+          previousDropGeoHash = Nothing,
+          driverTags = Just [NormalDriver, GoHomeDriverToDestination]
         }
```

Comment:
Why? Where? How? What?

Reply:
so this function is basically returning all the goHome Drivers which are going towards the destination so assigned these tags.